src/vaults/trading-apys/implementations/balancer-boost.js

```diff
@@ -17,15 +19,20 @@ const getBoostAPY = async (poolAddress, networkId) => {
   let apy = new BigNumber(0)
   for (let i = 0; i < tokens.length; i++) {
     const token = tokens[i]
-    const tokenWeight = tokenValues[token].div(tvl)
+    const tokenWeight = tokenValues[token]
+      ? tokenValues[token].div(tvl)
+      : tokenValues[token.toLowerCase()].div(tvl)
 
     let partApy
     if (types[i] == 'Aave') {
       partApy = await getAaveApy(poolBoostInfo.aaveTags[i])
+      partApy = networkId == '1' ? partApy.div(2) : partApy
```

Comment:
Looking at the numbers displayed at the AURA front page, it seems that they do not take the 50% cut in consideration. I am not sure if it's a mistake at their side or ours, but I think we want to display the same way they do, as we are competing with their displayed values. So let's remove the 50% cut here and we'll see if we want to change it in the future.

Reply:
Agree, I have fixed it!